app/models/dwc_occurrence.rb

```diff
@@ -0,0 +1,58 @@
+
+# A Darwin Core Record for the Occurence core.  Field generated from Ruby dwc-meta, which references
+# the same spec that is used in the IPT, and the Dwc Assistant.  Each record
+# references a specific CollectionObject or AssertedDistribution. 
+#
+# Important: This is a cache/index, data here are periodically (regenerated) from multiple tables in TW.  
+#
+# TODO: The basisOfRecord CVTs are not super informative.
+#    We know collection object is definitely 1:1 with PreservedSpecimen, however
+#    AssertedDistribution could be HumanObservation (if source is person), or ... what? if 
+#    its a published record.  Seems we need a 'PublishedAssertation', just like we model the data.
+#
+# DWC attributes are camelCase to facilitate matching
+# dwcClass is a replacement for the Rails reserved 'Class'
```

Comment:
Thanks for reminding me of this- there are a couple exceptions we have to catch like this.